aea/configurations/schemas/connection-config_schema.json

```diff
@@ -10,7 +10,6 @@
     "license",
     "aea_version",
     "class_name",
-    "config",
```

Comment:
I don't think `config` field in connection configuration should be required.

Reply:
It's used everywhere atm and seems little cost to keep at is? Are we sure making it optional has no unintended consequences?